in_container_scripts/mod_bashrc.sh

```diff
@@ -55,7 +55,10 @@ parse_git_branch() {
 if [[ ! -v DEPLOYMENT_DOCKER ]]; then 
   export PS1="\[\033[32m\]\h🐳 \[\033[36m\]\u@dev\[\033[m\]:\[\033[33;1m\]\w\[\033[m\]\$(parse_git_branch) $ "
 else
-  export PS1="\[\e[0;49;91m\]\h🐳 \[\033[36m\]\u@\e[0;49;91m\]deploy\[\033[m\]:\[\033[33;1m\]\w\[\033[m\] $ "
+  if [[ -f /src/deploy_version.txt ]]; then
```

Comment:
Do we really want drake-torch to know about this file?  Seems like that is a concern for higher up and not this repo.

Reply:
+1, or we might as well set it in `DEPLOYMENT_DOCKER`